src/firefly/js/templates/lightcurve/LcManager.js

```diff
@@ -273,30 +274,34 @@ export function* lcManager(params={}) {
 }
 
 
-function updateRawTableChart(timeCName, fluxCName, mission='') {
+function updateRawTableChart(timeCName, fluxCName, converterId) {
     var chartX = get(getChartDataElement(LC.RAW_TABLE), ['options', 'x', 'columnOrExpr']);
     var chartY = get(getChartDataElement(LC.RAW_TABLE), ['options', 'y', 'columnOrExpr']);
 
     if (chartX === timeCName && chartY === fluxCName) return;
 
     if (timeCName && fluxCName) {
 
-        const  title = (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other') )? 'Input Data':'';
+        const rawTable = getTblById(LC.RAW_TABLE);
+
+        //const  title = (mission && (mission.toLowerCase()==='wise' || mission.toLowerCase()==='other') )? 'Input Data':'';
+        const title = getConverter(converterId).showPlotTitle? rawTable.request.META_INFO.title :'';
```

Comment:
The intent of the converter is to provide dynamic data so that the core(common) logic does not need to know where/how the data came from.  So, if the logic is to display a plot title when available, shouldn't there be a converter.getPlotTitle().  Then, you display it when not empty.  But instead, it requires that the title comes from meta_info of the raw_table's request which is used as the table's title.  What if you want them to be different or you want to display something else?  Anyways, it's working, so you can leave it as is if you like.